src/com/carolinarollergirls/scoreboard/defaults/DefaultScoreBoardModel.java

```diff
@@ -189,14 +198,17 @@ public void stopJamTO() {
 				createSnapshot(ACTION_STOP_JAM);
 				setLabels(ACTION_START_JAM, ACTION_NONE, ACTION_TIMEOUT);
 				_endJam(false);
+				finishReplace();
```

Comment:
We should probably have unittests covering each of these new finishReplace calls

Reply:
I've extended the unittest that was already there instead of adding new ones.